src/action/action.yml

```
@@ -1,36 +1,36 @@
-      run: npm run build
-    - shell: bash
-      run: npm link
-    - shell: bash
-      run: |
-        code-audit analyze "${{ inputs.path }}" --format ${{ inputs.format }} --fail-on ${{ inputs.fail-on }} --out audit.md
-        code-audit grade "${{ inputs.path }}" --out grade.json
-        SCORE=$(jq -r '.score' grade.json)
-        GRADE=$(jq -r '.grade' grade.json)
-        echo "score=$SCORE" >> $GITHUB_OUTPUT
-        echo "grade=$GRADE" >> $GITHUB_OUTPUT
+name: "Code Audit TS"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)